### PR TITLE
fix call to msgpack

### PR DESCRIPTION
**Pull request by acxz**
This PR resolves https://github.com/ROCmSoftwarePlatform/AMDMIGraphX/issues/1602 by correcting the call to find_package for msgpack.

### CODE REVIEW

**Comment by pfultz2** [comment]
msgpack 3.3 doesn't provide a `msgpack-cxx` package, which is why its failing CI.

**Comment by acxz** [comment]
I see, I'm attempting to install migraphx with msgpack version 5.0.0

**Comment by acxz** [comment]
@pfultz2 I have updated the PR, can you restart the CI?

File: src/CMakeLists.txt [hunk 1]
```diff
@@ -251,7 +251,10 @@ find_package(PkgConfig)
 pkg_check_modules(SQLITE3 REQUIRED IMPORTED_TARGET sqlite3)
 target_link_libraries(migraphx PRIVATE PkgConfig::SQLITE3)
 
-find_package(msgpack REQUIRED)
+find_package(msgpackc-cxx REQUIRED)
```
Comment (pfultz2) [inline src/CMakeLists.txt:254]: `REQUIRED` needs to be removed on the first `find_package`.

Comment (acxz) [reply src/CMakeLists.txt]: Ah thx for the catch

Comment (acxz) [reply src/CMakeLists.txt]: Updated

**Review by pfultz2 — APPROVED** [review]

**Pull request merged**